docs/modules/ROOT/partials/getting-started/con-registry-overview.adoc

```diff
@@ -45,3 +45,6 @@ endif::[]
 * Compatibility with existing Confluent schema registry client applications
 
 * Operator-based installation of {registry} on OpenShift
+
+* Support for oidc based authentication
+
```

Comment:
Suggest spelling out the acronym:
"Support for OpenID Connect (OIDC) authentication"